server/app/views/admin/settings/AdminSettingsIndexView.java

```diff
@@ -234,7 +234,8 @@ private static DivTag renderIntInput(
         FieldWithLabel.number()
             .setFieldName(settingDescription.variableName())
             .setPlaceholderText("empty")
-            .setDisabled(settingDescription.isReadOnly());
+            .setDisabled(settingDescription.isReadOnly())
+            .setReadOnly(settingDescription.isReadOnly());
```

Comment:
Yeah it's kind of a silly to set both readonly and disabled, but it's a quick fix and to truly be done correctly we'd want to do [this bug](https://github.com/civiform/civiform/issues/5623). Given Greg's work on new controls for Northstar I'm not inclined to go to the extra effort since it will be being phased out.